explainer.md

```diff
@@ -147,7 +150,18 @@ The root elements of each page are automatically associated.
 
 Note that the order in which the transition elements are painted can be configured by UA and/or developer stylesheets using z-index.
 
-- [Open question](https://github.com/WICG/shared-element-transitions/issues/23): How should the default UA animation order these elements? And also handle a change in associated elements between the 2 pages. 
+- [Open question](https://github.com/WICG/shared-element-transitions/issues/23): How should the default UA animation order these elements? And also handle a change in associated elements between the 2 pages.
+
+### How are transition elements painted?
+During the transition a new stacking context (called uber-root) is created with the following hierarchy :
```

Comment:
I don't really like the name uber-root for official explainers. I don't have a better name, but maybe we can just not name it?

Reply:
I mostly did it for the diagram below to make the proposed hierarchy obvious. In general, we have a lot of naming in this explainer that needs to change. :)